decode.c

```diff
@@ -1844,7 +1845,7 @@ void log_regs(x86emu_t *emu)
 void check_data_access(x86emu_t *emu, sel_t *seg, u32 ofs, u32 size)
 {
   char **p = &emu->log.ptr;
-  static char seg_name[7] = "ecsdfg?";
+  static char seg_name[8] = "ecsdfg?";
```

Comment:
The original length is correct. It's not a string, just an array of chars. And the code clearly limits the access to
index 6.

I see no warning in gcc 10, which is usually quite picky. Maybe your compiler has a way to tell it that here not a
asciiz-string is meant.

If I add the 0 byte just to shut a warning down, I might get a warning about an unused byte by some other
compiler later.